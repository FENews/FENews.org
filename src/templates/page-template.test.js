import React from 'react';
import renderer from 'react-test-renderer';
import PageTemplate from './page-template';

describe('PageTemplate', () => {
  const props = {
    data: {
      markdownRemark: {
        html: '<p>test</p>',
        frontmatter: {
          title: 'test',
          description: 'test'
        }
      },
      site: {
        siteMetadata: {
          title: 'test',
          subtitle: 'test'
        }
      }
    }
  };

  it('renders correctly', () => {
    const tree = renderer.create(<PageTemplate {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
