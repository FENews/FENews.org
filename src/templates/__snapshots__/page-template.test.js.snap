// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTemplate renders correctly 1`] = `
<div
  className="layout"
>
  <StaticQuery
    render={[Function]}
  />
  <div
    className="page"
  >
    <div
      className="page__inner"
    >
      <h1
        className="page__title"
      >
        test
      </h1>
      <div
        className="page__body"
      >
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>test</p>",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
